tools/exchange_template/wrapper_file.tmpl

```diff
@@ -127,7 +127,7 @@ func ({{.Variable}} *{{.CapitalName}}) SubmitOrder(p pair.CurrencyPair, side exc
 
 // ModifyOrder will allow of changing orderbook placement and limit to
 // market conversion
-func ({{.Variable}} *{{.CapitalName}}) ModifyOrder(orderID int64, action exchange.ModifyOrder) (int64, error) {
+func ({{.Variable}} *{{.CapitalName}}) ModifyOrder(action exchange.ModifyOrder) (exchange.ModifyOrderResponse) {
```

Comment:
I think this is the last thing, this one need to be updated as well

Reply:
Legend 🌮